lib/generators/qa/install/templates/config/initializers/qa.rb

```diff
@@ -14,4 +14,8 @@
   # For linked data access, specify default language for sorting and selection.  The default is only used if a language is not
   # specified in the authority's configuration file and not passed in as a parameter.  (e.g. :en, [:en], or [:en, :fr])
   # config.default_language = :en
+
+  # When true, prevents ldpath requests from making additional network calls.  All values will come from the context graph
+  # passed to the ldpath request.
+  # config.limit_ldpath_to_context = true
```

Comment:
Add ability to determine whether ldpath will look outside the passed in context graph.